source/features/epochs/components/StakeDistributionRow.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
```

Comment:
@DeeJayElly Any intention why class component is used for this case specially as compared to other functional components?

Reply:
I changed it because i have been using Component approach for some other components. Its fixed now